### ISSUE TITLE
Nicehash miner hangs at starting excavator server

### ISSUE THREAD

**Issue by hariharanpalani** [nicehash/NiceHashMiner-Archived#1361]
Hi,
 I am trying to launch the nicehash miner version 2.0.1.5. It hangs at starting excavator server step. Can you please let me know any settings needs to be changed or it is really issue?

**Comment by interbiznw** [comment]
They were hacked and lost ~$60M.
https://www.reddit.com/r/NiceHash/comments/7i0s6o/official_press_release_statement_by_nicehash/

**Comment by F1Rumors** [comment]
That said, they are up and running again now, yet the excavator launch remains a problem

**Comment by Motherboard** [comment]
I experience it myself.. It started OK yesterday, but after a while the console window was gone and the GUI was showing as if it was mining, but the GPU was idle. After closing NiceHash Miner and starting it again it said it updates the excavator, and hangs on Starting Excavator Server (the process is using plenty of CPU, but nothing else is happening).

Tried to restart the computer - NiceHash Miner needed again to update the excavator, and it again hangs on Starting Excavator Server... 
  
Even complete uninstall and reinstall didn't change anything - it doesn't even get to the stage in which I can setup a worker again

**Comment by F1Rumors** [comment]
So we are clear, v2.0.1.6 has the same issue - you wait 30 minutes for it to "do something" when it ought to have stared the excavator and be useful already.  If I was a betting person, I would guess the excavator is not starting, and the methods that leverage it are just timing out without communicating the error.  Of course, I am not - I watched the processes as it started & I could see that no excavator process was successfully kicked off.

I also tried installing & executing with admin permissions, in case it was that simple. It was not.

**Comment by F1Rumors** [comment]
Right. Bit the bullet & took a closer look.  The excavator download is failing, because the anti-virus is eating the results of the unzip attempt, so the expansion is missing important files (like the executable).  You may just want to turn off scanning for a couple of minutes whilst starting the application... that will permit the expansion to work. Then you need to ensure that files under `C:\Users\<YOUR ACCOUNT>\AppData\Roaming\nhm2\bin\excavator_server` then/now `C:\Users\<YOUR ACCOUNT>\AppData\Roaming\nhm2\bin\excavator_cuda9.1` are excluded from the anti virus, or the excavator server will get randomly deleted whilst you are mining!

**Comment by Motherboard** [comment]
You're right, it was the antivirus, after whitelisting excavator it worked fine.

**Comment by josephpusateri** [comment]
Good lord!  I am so grateful to have found this thread... been dealing with this problem for a while.  Turning off scanning for a moment did the trick

**Comment by GianniKoch** [comment]
Hi I got the same issue, just turn your antivirus realtimescan off and restart your nicehash app, it will work, same for the v1 versions where your nicehashminer start exe deletes it selfs after you got an error that there are no system things... just turn of your anti virus and it will be fixed!
